content/Solutions/Optimizations/STIGCompliance.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "STIG Compliance"
+description: "Response to General Operating System STIG SRG for the TrueNAS appliance."
+weight: 35
+aliases:
+tags:
+ - ssh
+ - 2fa
+keywords:
+ - TrueNAS Security
+ - TrueNAS STIG Compliance
+---
+
+
+## TrueNAS Compliance
+TrueNAS falls into the category of an appliance with its own operating system as covered in [General Purpose Operating System SRG](https://www.stigviewer.com/stig/general_purpose_operating_system_srg/) findings.
+Through connection to Active Directory, TrueNAS also complies with the [Active Directory Domain Security Technical Implementation Guide SRG](https://www.stigviewer.com/stig/active_directory_domain/) findings related to authentication and access controls for user, group, and systems.
+
+## Customizing TrueNAS Security Options for STIG Compliance
+Many areas of compliance with the STIG SGR findings are automatically addressed through the TrueNAS kernel and middleware, but some are optional settings and features in the TrueNAS UI administration users customize to suit individual use cases and security policies.
+This article details customizable settings to accomplish a security-hardened systems for STIG compliance.
+
+## Install TrueNAS
+Existing TrueNAS systems can upgrade to the latest release through the UI.
+Earlier releases of TrueNAS can upgrade by following the established migration paths, or might need to clean-install the latest release of SCALE using the <file>iso
+</file>. Refer to documentation on upgrading existing systems found [here]({{< relref "/content/TrueNASUpgrades/_index.md" >}}).
+
+If side-grading (migrating) from an earlier release to the latest TrueNAS release, follow guidance in the [Preparing to Migrate]({{< relref "/SCALE/GettingStarted/Migrate/MigratePrep.md" >}}) article before beginning the migration.
+
+If clean-installing TrueNAS for the first time, follow guidance in the [Installation Instructions]({{< relref "/content/SCALE/GettingStarted/Install/_index.md" >}}) articles.
+
+### Prerequisites
+Installing TrueNAS on servers other than iXsystems-provided systems, should read and follow guidance in the [TrueNAS Hardware Guide]({{< relref "SCALEHardwareGuide.md" >}}).
+
+## Configuring TrueNAS
+After installing TrueNAS, users must complete the initial configuration of network, storage, users, sharing, and data backup solutions. See [Configuration Instructions]({{< relref "/SCALE/gettingstarted/configure/_index.md" >}}) for more information.
+
+### Administration Accounts
+TrueNAS creates the root user and an administration user at installation.
+Some releases of TrueNAS might only have **root** as the default administration user, while other releases have either the **admin** or **truenas_admin** as the default user.
+
+TrueNAS systems with the **root** user, or either the **admin** or **truenas_admin** user should create a new administration user with full control privileges and assign a complex password that you should change on a frequent basis.
```

Comment:
The government standards for passwords just changed (within the last couple weeks.)  They don't suggest changing passwords frequently any longer.  Might want to change that to something like "assign a complex password and follow current password guidelines for managing it" or something.

Reply:
change made in next commit